keys.json

```diff
@@ -1,55 +1,103 @@
 {
-    "ace":           { "skip": true, "//": "API randomly fails and returns a HTML error" },
+    "ace":           { "skipMethods": { "fetchOrderBook": "needs reversion of amount/price", "fetchL2OrderBook": "same" } },
     "alpaca":        { "skipWs": true },
-    "bkex":          { "skip": true },
-    "binance":       { "skipWs": true, "httpProxy": "http://51.83.140.52:11230" },
-    "binanceusdm":   { "skipWs": true, "httpProxy": "http://51.83.140.52:11230" },
+    "ascendex":      { "skipWs": true, "skipMethods": { "fetchTickers": "bid might be 0", "fetchTicker": "same"} },
+    "bequant":       { "skipWs": true, "skipMethods": { "fetchTickers": "close might have null value", "fetchTicker": "same"}  },
+    "bigone":        { "skipMethods": { "fetchCurrencies": "info key is missing", "fetchTickers": "open value can be 0", "fetchTicker": "same"} },
+    "binance":       { "skipWs": true, "httpProxy": "http://51.83.140.52:11230", "skipMethods": { "loadMarkets": "expiry not set for future markets", "fetchTickers": "some symbols i.e. BTC/USD4 might have bid-ask value 0", "fetchTicker": "same" } },
```

Comment:
this is just too verbose and hard to read, I don't think having the reason per method is necessary.
Can we replace it with a flatter structure?
Example:
```
"skipMethods":["fetchOrderBook", "fethTicker"]
```

Reply:
why should we vouch for worse for no reason? we don't read them everyday, and I think it will not hurt, and is more accurate/informatic. otherwise, what does `"skipMethods":["fetchOrderBook", "fethTicker"]` mean? why we do skip fetch-orderbook or fetchTicker there? 
in the end , actually, all these current values are temporary to help the current PR to pass, and after that, those issues will need to be skipped inside implementations, so the keys.json won't be too large.

Reply:
I understand, but this is not a clean structure and if you have a lot of exchanges with these annotations the file will be very oddly shaped and unredable

Reply:
right, and I think we wouldnt have many such annotations. as I said, even these current notes will be much reduced after merge, as then we  will fix most of those issues quickly in implementations, or add tolerance in tests (50%+ of them are comparisons to 0). so, i look at this optimistically.

Reply:
> even these current notes will be much reduced after merge
One more reason not to have them IMO. This file should be straightforward to navigate for us and for the users because they might be copying it from the repo to add the keys and proxies.

Reply:
ok, we can create a separate json file to write these skiped things so? maybe `skip.json` or whatever.

Reply:
i've pushed changes needed for that new design (with separated skip settings)

Reply:
@ttodua yes probably it's better to place it in a different file but if we're doing that I think we should use another approach. Instead of defining it per exchange, we can do the other way around no?

```Javascript
{
 "skip":[],
 "skipWs": [],
 "fetchTicker": []  
}
```

I think we would avoid some duplicated code and it's probably easier to check

Reply:
the reason 'per exchange' is better imo, that it's specific. when mostly working (any dev) per specific exchange, he needs to fully ground around it, so scattering through different parts might not be ideal imo. I think it's quite precise to look at specific exchange and you will see what does that exchange lacks/what issues it has.

yeah, donwside is that there are some duplicated code, but again, i dont see it much a problem at this moment, we will soon fix those things inside implementations and the content will be less.

Reply:
@ttodua Fair enough, but let's avoid this structure `{ "fetchTickers": "bid might be 0"}`, I would rather have something like `{"fetchTickers":true, reason:"Very succinct reason here}"

Reply:
why to complicate the objects? also, it will not be ok, as there might be multiple ones, so how to write multiple `reason`s?
```
skipMethods: { "fetchTicker":  , "reason": "XXXXX", "fetchOrderBook": .., "reason": "YYYY" }
```